src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -77,6 +82,19 @@ export class SmartSnippet extends Component {
     });
   };
 
+  /**
+   * The options for the SmartSnippet
+   * @componentOptions
+   */
+  static options: ISmartSnippetOptions = {
+    /**
```

Comment:
Here's a suggestion for the whole string:
```
The maximum height an answer can have in pixels. 
Any part of an answer exceeding this height will be hidden by default and expendable via a "show more" button. 
Default value is `250`.
```

Reply:
Thanks! Using it.